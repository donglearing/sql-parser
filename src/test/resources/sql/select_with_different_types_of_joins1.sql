select t.a, f.b from T as t inner join F as f on t.a = f.b left join H as h on f.c = h.c natural left join G as g cross join I as i union join J as j natural inner join K as k;