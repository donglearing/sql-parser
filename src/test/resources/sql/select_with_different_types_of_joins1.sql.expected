SELECT t.a, f.b FROM T AS t INNER JOIN F AS f ON t.a = f.b LEFT JOIN H AS h ON f.c = h.c NATURAL LEFT JOIN G AS g CROSS JOIN I AS i UNION JOIN J AS j NATURAL INNER JOIN K AS k